src/psql/migrations/UAT-CSTAR/fa/V1__fa.pre.sql

```diff
@@ -70,22 +70,33 @@ CREATE SCHEMA fa_transaction;
 
 ALTER SCHEMA fa_transaction OWNER TO ddsadmin;
 
+--
+-- Name: fa_error_record; Type: SCHEMA; Schema: -; Owner: ddsadmin
+--
+
+CREATE SCHEMA fa_error_record;
```

Comment:
@daniloMessinaSia it would be better to use a different flyway migration - maybe `V6__xxxx` - for all the proposed changes, otherwise I have to drop the DB and reapply all migrations already present

Reply:
ok, i will create a new script for all changes